src/iuse.cpp

```diff
@@ -6485,22 +6498,23 @@ const auto trap_name_at = []( const tripoint &point )
 
 std::string get_field_description_at( const tripoint &point )
 {
-    static const std::unordered_set<field_id> covered_in_affix_ids = {
+    typedef std::unordered_set<field_id, std::hash<int>> Field_id_set;
+    static const Field_id_set covered_in_affix_ids = {
```

Comment:
This is that i have found related to error: https://stackoverflow.com/a/29618545